src/server/StudioManagementSystem.Infrastructure/Repositories/GroupApplicationRepository.cs

```diff
@@ -48,11 +48,47 @@ public async Task<bool> AddGroupApplication(InvitationDto dto, CancellationToken
 
         return true;
     }
+
+    public async Task<bool> ManageGroupInvitation(ManageInvitationDto dto, CancellationToken ct)
```

Comment:
We can simplify this method if it only handles rejection. This would line up with the frontend logic a well.